### ISSUE TITLE
[FEATURE] Add language switcher to Dashboard

### ISSUE THREAD

**Issue by datnguyen293** [opensearch-project/i18n-plugin#16]
**Is your feature request related to a problem?**
Currently the translation is added in the `config/opensearch_dashboards.yml` file, the `i18n-plugin` is only to maintain the list of translations without ability for user to change/switch the language runtime.

**What solution would you like?**
Add the JS function or query string to the URL to specify the language. The `i18n-plugin` is able to load the according translation file when a new locale/language is specified.

**What alternatives have you considered?**
Adding a language switcher component to `i18n-plugin` or the dashboards would be nice to have.

**Do you have any additional context?**
No.

**Comment by joshuarrrr** [comment]
I agree that it would be a nice feature to have - I'm not sure a query string is the right way to go, but I could definitely imagine adding it to the user settings.

**Comment by smowky** [comment]
+1 
would be nice feature to have!
